packages/components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -36,11 +36,11 @@ const TooltipContainer = styled(motion.div)<{
     $isLarge: boolean;
     $isWithHeader: boolean;
 }>`
-    background: ${({ theme }) => theme.backgroundNeutralBold};
-    color: ${({ theme }) => theme.textOnPrimary};
+    background: ${palette.darkGray300};
+    color: ${palette.lightWhiteAlpha1000};
     border-radius: ${borders.radii.sm};
     text-align: left;
-    box-shadow: ${({ theme }) => theme.boxShadowElevated};
+    border: solid 1.5px ${palette.darkGray100};
```

Comment:
Yes it's correct. These colors shouldn't have light/dark mode

Reply:
Maybe lets add a comment into the code